src/main/resources/model/bibframe.json

```diff
@@ -0,0 +1,169 @@
+{
+  "name": "bibframe",
+  "eventBodyJavaClass": "org.folio.search.domain.dto.Bibframe",
+  "languageSourcePaths": [ "$.languages" ],
+  "fieldTypes": {
+    "suppress": {
+      "searchTypes": [ "facet", "filter" ],
+      "index": "bool",
+      "default": false
+    },
+    "visibleSuppress": {
+      "searchTypes": [ "facet", "filter" ],
+      "index": "bool",
+      "default": false,
+      "showInResponse": [ "search" ]
+    }
+  },
```

Comment:
I can't find where these defined types are used

Reply:
Thank You. Removed.
@psmagin